CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.1.0 (2019-04-09)
+
+- Added ability to place items on the right side of toolbar components. [#18](https://github.com/blackbaud/skyux-layout/pull/18)
```

Comment:
"...of the toolbar component."    Thoughts?

Reply:
Done